.github/workflows/npm-test.yml

```diff
@@ -0,0 +1,26 @@
+name: Test
+
+on:
+  pull_request:
+    branches:
+      - master
+      - stable*
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [11.x]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: npm install
```

Comment:
This is stupid, but can I ask if we can have the same case for the tests names? :see_no_evil: 
![image](https://user-images.githubusercontent.com/14975046/73629444-6d684000-4653-11ea-8fc6-2726277b89c3.png)

```suggestion
      - name: Install dependencies
```

Reply:
Fine with me, I also wondered how I should name the tests 😃

Reply:
I standardised it for server/contacts & photos so far :woman_shrugging: 
https://github.com/nextcloud/photos/tree/master/.github/workflows

Reply:
Then I guess we need another PR for nextcloud-vue 😉